packages/firestore/test/unit/local/remote_document_cache.test.ts

```diff
@@ -370,9 +373,9 @@ function genericRemoteDocumentCacheTests(
 
   it('can remove document', () => {
     return cache
-      .addEntries([doc(DOC_PATH, VERSION, DOC_DATA)])
+      .addEntries([doc(DOC_PATH, VERSION, DOC_DATA)], version(VERSION))
       .then(() => {
-        return cache.removeEntry(key(DOC_PATH));
+        return cache.removeEntry(key(DOC_PATH), version(VERSION + 1));
```

Comment:
`VERSION + 1` implies that this is meaningful, but it's not, right?

Reply:
Well. It happens later so it should have a later read time... but yes, it could also be VERSION - 1 and everything would be fine and dandy :/